cetmix_tower_server/models/cx_tower_scheduled_task.py

```
@@ -0,0 +1,257 @@
+            # WARNING: Explicit commit!
+            # This commit is made **only** when called from cron (context["from_cron"]).
+            # Reason: To atomically reserve scheduled tasks by setting is_running=True,
+            # so that only one cron worker processes each task, even if multiple workers
+            # pick up the cron job at the same time. Without this commit, the change
+            # would not be visible to other transactions until the end of the cron
+            # transaction, leading to a race condition and possible double execution.
+            # Explicit commits are strongly discouraged in Odoo business logic and
+            # should be used only with clear justification and in strictly controlled
+            # contexts (like this cron scenario). Never add this commit for general
+            # business flows!
+            self.env.cr.commit()  # pylint: disable=invalid-commit
```

Pylint (3.3.7)

[warning] Unknown option value for 'disable', expected a valid pylint message and got 'invalid-commit'

(W0012)